### ISSUE TITLE
Honor 8 lite Network/sim continuouly disconnects and reconnects

### ISSUE THREAD

**Issue by amrithmmh** [phhusson/treble_experimentations#1822]
Hi,
I tried almost all the versions of GSI of Lineage starting from android 8.1 (No network issue) to ASOP 11 and lineage 18 .
problem is the network gets connected (LTE recommended mode) and then after a second it disconnects and shows no network this keeps repeating.

**Comment by mischa85** [comment]
This is still happening on my P8lite 2017 PRA-LX1 on phhusson AOSP and anything derived from it.

Tried with stock and vanilla kernels.

Huawai P8lite PRX-LX1 Prague_L31_HN hw eu
EMUI 8.0.0.400 C43

[p8litephhussoncellularnotworking.zip](https://github.com/phhusson/treble_experimentations/files/7829737/p8litephhussoncellularnotworking.zip)
2

**Comment by phhusson** [comment]
If I'm not mistaken, your logs have been taken with `adb logcat -b radio` right?
You need to give me logs created with `adb logcat -b all`

**Comment by mischa85** [comment]
My bad. As these logs are very very verbose I wanted to keep only the relevant parts. Here are the complete logs for EMUI 8.0.0.400 stock ROM which works, and Phhusson AOSP 11.0 v313 which keeps asking for PIN (I tried 3 times while logging).

Stock logging:
[simcardinsertpincodestock.zip](https://github.com/phhusson/treble_experimentations/files/7831480/simcardinsertpincodestock.zip)

Phhusson logging:
[simcardinsertpincodephhusson.zip](https://github.com/phhusson/treble_experimentations/files/7831481/simcardinsertpincodephhusson.zip)

**Comment by phhusson** [comment]
If you disable PIN on sim card, is the behavior the same?

**Comment by mischa85** [comment]
Yes it's the same. It just tries to connect all the time and fails.

`RIL_GU  : unsolicitedResetCommand,Have receive three modem reset`

**Comment by amrithmmh** [comment]
Yes . Any work around. Previous build had a workaround for pra-al00x 
setting sim lock , the system will ask for sim unlock pin multiple times but sometime it becomes stable. 
But with A11 the workaround is not working for me .
Everything else works fine
https://forum.xda-developers.com/t/rom-huawei-11-lir-lineage-18-1-for-huawei-device.4379105/post-86216091
